lib/hades/mentorships/mentor.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Hades.Mentorships.Mentor do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias Hades.Mentorships.Mentor
+
+  @allowed_skill_areas ~w(Backend Frontend DevOps UX/UI Mobile Fullstack)
+
+  schema "mentors" do
+    field :is_active, :boolean, default: false
+    field :max_mentorships, :integer
+    field :skill_areas, {:array, :string}
+    field :user_id, :id
```

Comment:
I think we could use `belongs_to :user, Hades.Accounts.User` instead. Also: in the user schema we should have `has_one :mentor, Hades.Mentorships.Mentor`

Reply:
makes sense, let's do that.